msg/InferenceInfo.msg

```diff
@@ -0,0 +1,23 @@
+# Provides meta-information about a visual pipeline.
+#
+# This message serves a similar purpose to sensor_msgs/CameraInfo, but instead
+#   of being tied to hardware, it represents information about a specific
+#   computer vision pipeline. This information stays constant (or relatively
+#   constant) over time, and so it is wasteful to send it with each individual
+#   result. By listening to these messages, subscribers will receive
+#   the context in which published vision messages are to be interpreted.
```

Comment:
this will need to be updated based on the caching/latched/etc. discussion in the README.

Reply:
> A message truly like CameraInfo - it is not expected to change.

Wait - I think it's the other way around. CameraInfo gets published with each published info (at least that's the way it is in ROS1, has that changed?). While it's true that it doesn't change in 99% of cases, the way that the message is designed and handled in ROS1 is that you're supposed to create a time synchronizer and always subscribe to the CameraInfo + Image pair.

BTW, the only existing use case that I know of where the CameraInfo changes is when you run [camera_calibration](https://wiki.ros.org/camera_calibration) and click "COMMIT", which will call a service on the camera driver to update its published CameraInfo. That's really an edge case.